src/genie/libs/parser/iosxe/tests/ShowCtsRoleBasedPermissions/cli/equal/sample_output_expected.py

```diff
@@ -0,0 +1,10 @@
+expected_output = {
+    "indexes": {
+        1: {
+            "action_policy": "Permit",
+            "action_policy_group": "IP-00"
+        },
+        "monitor_dynamic": False,
+        "monitor_configured": False
```

Comment:
even from parsed output, we could guess what they are. but I assume this name is very short. and doesn't tell this is for RBACL related. please consider updating schema with better key name.